osbs/cli/main.py

```diff
@@ -399,21 +385,13 @@ def cmd_build_logs(args, osbs):
     build_id = args.BUILD_ID[0]
     follow = args.follow
 
-    if follow and args.from_docker_build:
-        print("Can't use --follow and --from-docker-build. "
-              "Logs from docker build are part of metadata of a already built image.")
+    logs = osbs.get_build_logs(build_id, follow=follow,
+                               wait_if_missing=args.wait_if_missing,
+                               decode=True)
```

Comment:
Any concerns with using `decode=True` if follow if `False` ?

Reply:
ahh ok I'll put it bellow that if as it should be

Reply:
mmm this logic wasn't changed, should those 2 things be conflicting?

Reply:
Yes, you're right. Logic is the same.

I vaguely remember a while back there was an issue with follow and decode.